### ISSUE TITLE
User-oriented titles

### ISSUE THREAD

**Issue by jbs1** [OpenMath/OM3#10]
_migrated from Trac, where originally posted by **polx** on 6-Nov-2007 11:11pm_

The symbol name is a URI hence is neither translatable nor really  
human-readable. The human-name of a symbol, that is its title, can be  
very often used in rendered OpenMath. 

Multilingual versions should be   available.

**Comment by jbs1** [comment]
_migrated from Trac, where originally posted by **kohlhase** on 8-Nov-2007 12:45pm_

I agree that a title or short description of a symbol would be very helpful, and yes multilinguality would also be nice. Do I understand you correctly that you are suggesting a field <Title> as a sibling to <Name>? Multilinguality could be reached by   giving <Title> an xml:lang field and allowing Title\* in the content Model. 

But maybe we should rename the current <Name> to id and use <Name> for what I called <Title> above. 

At the MathML F2F we discussed something very related; see the discussion in Issue 11
